src/library/Uno.Material/Styles/Controls/BottomNavigationBarItem.xaml

```diff
@@ -71,6 +71,18 @@
 										<Setter Target="Label.Foreground"
 												Value="{StaticResource BottomNavLabelFocusColorBrush}" />
 									</VisualState.Setters>
+
+									<!-- TODO: Investigate why ColorAnimation isn't working -->
```

Comment:
Open github issue please!

Reply:
I'll make a note of this right now and link to this particular code.